### PR TITLE
Adding Analytics context data

### PR DESCRIPTION
**Pull request by ninaceban**
Please link to the issue #…
https://jira.corp.adobe.com/browse/PLAT-140711

### CODE REVIEW

File: extensions/adobe/experience/analytics/context-data.schema.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/context-data",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "context-data",
+  "type": "object",
+  "description": "A free form field where Analytics context data will be captured.",
+  "definitions": {
```
Comment (mjasrotia) [inline extensions/adobe/experience/analytics/context-data.schema.json:13]: This definition should also have a "properties" object which will further have the map object you want to define.
Example:
`  "definitions": {
    "context-data-datatype": {
      "properties": {
        "xdm:<Map Object Name>": {
          "type": "object",
          "meta:xdmType": "map",
          "additionalProperties": {
            "type": "string",
			"description": "Your Description"
          }
        }
      }
    }
  }
`
an existing example you can refer is 
https://github.com/adobe/xdm/blob/master/components/fieldgroups/shared/identitymap.schema.json

Please make the changes.

Comment (ninaceban) [reply extensions/adobe/experience/analytics/context-data.schema.json:13]: thank you! I'll add the changes!

File: extensions/adobe/experience/analytics/context-data.schema.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "context-data-datatype": {
+      "properties": {
+        "xdm:context-data-datatype": {
```
Comment (mjasrotia) [inline extensions/adobe/experience/analytics/context-data.schema.json:16]: We do not support "-" in the property names. Also, it seems like you are creating a map object to send context data so having a suffix of "datatype" in the object name would not make sense. I would suggest to change the property name to "xdm:contextData"

File: extensions/adobe/experience/analytics/context-data.schema.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/context-data",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "context-data",
```
Comment (mjasrotia) [inline extensions/adobe/experience/analytics/context-data.schema.json:10]: Let's change this to "Context Data"

File: extensions/adobe/experience/analytics/context-data.schema.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "type": "object",
+  "description": "A free form field where Analytics context data will be captured.",
```
Comment (mjasrotia) [inline extensions/adobe/experience/analytics/context-data.schema.json:12]: Can we add more information in this description regarding the map object and the examples of data you would typically send in this field ?

File: extensions/adobe/experience/analytics/context-data.schema.json (hunk 2 above)
Comment (ninaceban) [reply extensions/adobe/experience/analytics/context-data.schema.json:16]: thank you, I'm adding that

**Pull request merged**